content/guides/tanzu-labs-engineer-faq/index.md

```diff
@@ -0,0 +1,364 @@
+
+Others may struggle with team code ownership for several reasons:
+
+* Shifting psychological ownership may require a corresponding shift in an individual's identity. Psychological ownership refers to "the feeling of possessiveness and of being psychologically tied to an object." Psychological ownership occurs when the object becomes part of the psychological owner's identity. Psychological ownership answers the question, "What do I feel is mine?"
+* Developers may struggle to transition to a caretaker mindset. An engineer struggled to describe the developer's relationship with the code on a very challenging project and settled on the caretaker metaphor: "Sometimes I feel like a janitor to [the code base]. Maybe "caretaker" would be better. Yeah, probably caretaker. I feel like a janitor just cleans up messes, but a caretaker makes things better."
+* A developer may be distraught at "seeing my work slowly removed from the app."
+* A developer may feel belittled when seeing their work changed or fixed "behind their back"
+* Developers can no longer take pride in functionality that they exclusively develop.
+* Existing knowledge silos, which hinder team code ownership, may be slow to break down.
+
+New hires struggling with the transition slowly realize that "Someone else is going to take over and they're going to do fine. I can move onto something else and that's okay." They recognize the lack of long-term individual authorship, learn to expect their code to be transitory, develop trust in their teammates and thus loosely hold personal contributions. "The code that I write today may be in the code base for a little while, and it will evolve into something better." Eventually, they experience the benefits of a collaborative environment: "People are a lot more flexible all across the board, with changing things or accepting feedback or collaborating," and the team can say "Hey, this is our code!"
+
+Shifting from individual to team code ownership may require multiple and complementary practices to actively remove knowledge silos. In this case, daily pair rotation helped combat knowledge silos. Moreover, for developers with strong individual ownership tendencies, sharing ownership first with a small group (where trust and communication come easier) may help. One engineer uses improvisation and collaboration games to help teams practice letting go of control, trusting the team, and learning to be pleasantly surprised by what emerges.
+
+See:[ https://www.researchgate.net/publication/301612260_Practice_and_Perception_of_Team_Code_Ownership](https://www.researchgate.net/publication/301612260_Practice_and_Perception_of_Team_Code_Ownership)
+
+## Q: Why Test Driven Development (TDD)? (Also, Why Do We Refactor?){id="tdd"}
+
+Consider these two graphs of team speed over time, and ask yourself which team you want to be on:
+
+![Fast at first](images/image3.jpg)
+
+
+![Go fast forever](images/image5.jpg)
```

Comment:
```suggestion
![Go fast forever](images/image3.jpg)
```

These two graphs should be swapped. The alt text matches the way these graphs are referenced below, but the images used are the reverse.

Reply:
It was called out that the TDD section was duplicated / covered by existing content on the site. I changed this section to refer to those other articles. Feedback welcomed regarding whether that works.